### ISSUE TITLE
[Designer][Bug 40020365](https://microsoft.visualstudio.com/OS/_workitems/edit/40020365): [ShowCardWizard.JSON]: "Please provide the following information" text is not associated with the first expandable button.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7610]
### Problem Description

[Bug 40020365](https://microsoft.visualstudio.com/OS/_workitems/edit/40020365): [ShowCardWizard.JSON]: "Please provide the following information" text is not associated with the first expandable button.

### Card JSON

```JSON
[Bug 40020365](https://microsoft.visualstudio.com/OS/_workitems/edit/40020365): [ShowCardWizard.JSON]: "Please provide the following information" text is not associated with the first expandable button.
```
